apps/Gruntfile.js

```diff
@@ -488,4 +488,5 @@ module.exports = function(grunt) {
 
   process.env.mocha_grep = grunt.option('grep') || '';
   process.env.mocha_debug = grunt.option('debug') || '';
+  process.env.mocha_entry = grunt.option('entry') || '';
```

Comment:
Right now if I grep for a single mochaTest, we'll still build the bundle.js for everything. This allows us to specify an entry point and limit how much gets bundled, leading to a slightly faster test run.
```
grunt mochaTest --grep=addScreenIfNecessary --entry=./test/applabTest.js